packages/cli/src/db.ts

```diff
@@ -72,6 +72,7 @@ export async function init(): Promise<void> {
 			});
 		}
 
+		ErrorReporter.error(error);
```

Comment:
This function is called in `BaseCommand.init`, which calls `exitWithCrash` in case of an error. That already has a call to `ErrorReporter.error`, so it should be covered already.

The caveat here is that we only capture these error in the app _startup_, but not after it. We can't really map the errors either as they can happen everywhere where a DB query is made. Or we would probably have to add the support into our `typeorm` fork. Shouldn't generic `Error`s be already reported?